sql/files/defaultdata/compare/compare.cc

```diff
@@ -72,33 +72,10 @@ FILE *openfeedback(const char *feedbackdir, const char *feedback, const char *wh
 	return res;
 }
 
-/* Test two numbers for equality, accounting for +/-INF, NaN and
- * precision. Float f2 is considered the reference value for relative
- * error.
- */
-int equal(flt f1, flt f2, flt float_abs_tol, flt float_rel_tol)
-{
-	flt absdiff, reldiff;
-	/* Finite values are compared with some tolerance */
-	if ( std::isfinite(f1) && std::isfinite(f2) ) {
-		absdiff = fabsl(f1-f2);
-		reldiff = fabsl((f1-f2)/f2);
-		return !(absdiff > float_abs_tol && reldiff > float_rel_tol);
-	}
-	/* NaN is equal to NaN */
-	if ( std::isnan(f1) && std::isnan(f2) ) return 1;
-	/* Infinite values are equal if their sign matches */
-	if ( std::isinf(f1) && std::isinf(f2) ) {
-		return std::signbit(f1) == std::signbit(f2);
-	}
-	/* Values in different classes are always different. */
-	return 0;
-}
-
 const char *USAGE = "Usage: %s judge_in judge_ans feedback_dir [options] < team_out";
 
 int main(int argc, char **argv) {
-	if(argc < 4) {
+	if (argc < 4) {
```

Comment:
Nitpick: could you make these unrelated layout changes in a separate commit?

Reply:
Sure, done! :smile: